App.tsx

```diff
@@ -70,7 +74,20 @@ if (Sentry?.init) {
   }
 }
 
-const queryClient: QueryClient = new QueryClient();
+const queryCache: QueryCache = new QueryCache();
+
+const queryClient: QueryClient = new QueryClient({
+  queryCache: queryCache,
+  defaultOptions: {
+    queries: {
+      cacheTime: 1000 * 60 * 60 * 24, // 24 hours
```

Comment:
Why cache in memory for 24 hours when we are *also* caching to storage?

How much ram do we use keeping all these query results loaded?

Reply:
also, I like the idea of setting a global policy but then it'd be good to remove all the explicit settings on each query.

Reply:
React Query will hydrate and de-hydrate from the async persister for it's storage, so the footprint will be roughly double, but it's *only* the textual content that comes back from API calls. The asset caching is *not* in this cache. Async storage itself is limited to something like 5MB, and it's stored in some weird plethora of places based on the [platform](https://react-native-async-storage.github.io/async-storage/docs/advanced/where_data_stored),